.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,34 @@
+name: "\U0001F680 Feature request"
+description: Suggest an idea for the Israel Hiking Map project
+labels: ["feature-request"]
```

Comment:
Enhancement instead?

Reply:
I prefer the more general "idea", which we can refine later.
However, if you prefer "enhancement" we'll go with that.

Reply:
I'm using this label now to indicate something that is not a bug. I hope this is what it means, right? i.e. when someone selects feature it will add that label to the issue that is being opened, right?

Reply:
You're right. I looked at the wrong line of code!